src/markdown-pages/supported-platforms.md

```diff
@@ -0,0 +1,143 @@
+|Ubuntu 16.04|<i class="fa fa-star-o hotspot" aria-hidden="true"></i> <i class="fa fa-star-o openj9" aria-hidden="true"></i>|<i class="fa fa-star-o hotspot" aria-hidden="true"></i> <i class="fa fa-star-o openj9" aria-hidden="true"></i>|<i class="fa fa-star-o hotspot" aria-hidden="true"></i> <i class="fa fa-star-o openj9" aria-hidden="true"></i>||<i class="fa fa-star-o hotspot" aria-hidden="true"></i>|
+|Ubuntu 18.04|<i class="fa fa-star-o hotspot" aria-hidden="true"></i> <i class="fa fa-star-o openj9" aria-hidden="true"></i>|<i class="fa fa-star-o hotspot" aria-hidden="true"></i> <i class="fa fa-star-o openj9" aria-hidden="true"></i>|<i class="fa fa-star-o hotspot" aria-hidden="true"></i> <i class="fa fa-star-o openj9" aria-hidden="true"></i>||<i class="fa fa-star-o hotspot" aria-hidden="true"></i>|
+
+**Note**: Any other Linux distributions that have a minimum glibc version 2.12 on x86 or version glibc version 2.17 on other architectures are expected to function without problems.
+
+|Windows|x32|x64|
+|--- |--- |--- |
+|Windows 7 SP1|<i class="fa fa-star-o hotspot" aria-hidden="true"></i>|<i class="fa fa-star-o hotspot" aria-hidden="true"></i> <i class="fa fa-star-o openj9" aria-hidden="true"></i>|
+|Windows 8|<i class="fa fa-star-o hotspot" aria-hidden="true"></i>|<i class="fa fa-star-o hotspot" aria-hidden="true"></i> <i class="fa fa-star-o openj9" aria-hidden="true"></i>|
+|Windows 8.1|<i class="fa fa-star-o hotspot" aria-hidden="true"></i>|<i class="fa fa-star-o hotspot" aria-hidden="true"></i> <i class="fa fa-star-o openj9" aria-hidden="true"></i>|
+|Windows 10|<i class="fa fa-star-o hotspot" aria-hidden="true"></i>|<i class="fa fa-star-o hotspot" aria-hidden="true"></i> <i class="fa fa-star-o openj9" aria-hidden="true"></i>|
+|Windows Server 2012|<i class="fa fa-star-o hotspot" aria-hidden="true"></i>|<i class="fa fa-star-o hotspot" aria-hidden="true"></i> <i class="fa fa-star-o openj9" aria-hidden="true"></i>|
+|Windows Server 2012 R2|<i class="fa fa-star-o hotspot" aria-hidden="true"></i>|<i class="fa fa-star-o hotspot" aria-hidden="true"></i> <i class="fa fa-star-o openj9" aria-hidden="true"></i>|
+|Windows Server 2016|<i class="fa fa-star-o hotspot" aria-hidden="true"></i>|<i class="fa fa-star-o hotspot" aria-hidden="true"></i> <i class="fa fa-star-o openj9" aria-hidden="true"></i>|
+
+|macOS|x64|
+|--- |--- |
+|OSX 10.10+|<i class="fa fa-star-o hotspot" aria-hidden="true"> <i class="fa fa-star-o openj9" aria-hidden="true">|
+
+|AIX|ppc64|
+|--- |--- |
+|AIX 7.1 TL4|<i class="fa fa-star-o openj9" aria-hidden="true"></i>|
+|AIX 7.2|<i class="fa fa-star-o openj9" aria-hidden="true"></i>|
+
+</div>
+
+<div class="home-links" style="margin-top: 3rem;">
```

Comment:
We can probably add [MDX](https://mdxjs.com/) support and extract this into a `<HomeLinks />` component.

Reply:
agreed, let's get this merged and then I'll work on refactoring it.